test/e2e/specs/onboarding/setup__build.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * @group calypso-pr
+ */
+
+import { DataHelper, SecretsManager, StartSiteFlow, TestAccount } from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Site Build with "Something else" category' ), () => {
```

Comment:
**question**: This suite (`Site Build with "Something else" category`) and the earlier one (`FTME: Build with "Something else" category`) both appear to do _very_ similar things.

One flow uses `defaultUser` and bypasses the new site signup, while the other uses `simpleSiteFreePlanUser` and perform a site signup. Other than that, everything between the two flows (down to the theme selected) appear to be identical. As a result there is significant amount of duplicated coverage between these four top-level flows.

Could we reconsider and limit the number of flows to a more manageable level?

Reply:
My original thought is one is for PR and the other is for release, but I agree that we only need to keep one. So, I'll remove `FTME: Build with "xxx" category` tests and only leave this suite, `Site Build with "Something else" category`.